src/containers/Profile/components/EditForm.js

```diff
@@ -655,34 +657,43 @@ const EditForm = ({
       )}
     />
 
-    <InputGroup>
-      <Field
-        name="birthday.month"
-        render={({ input, meta }) => (
-          <FormField fullWidth>
-            <Label
-              htmlFor={input.name}
-              text="Birthday (month)"
-              error={meta.touched && get(meta, 'error.length', 0) > 0}
+    <FormSpy subscription={{ values: true }}>
+      {({ values: { birthday } }) => {
+        const month = birthday && birthday.month.value;
```

Comment:
Probably safer to say: `get(birthday, 'month.value')` (ie. what if birthday.month is `null` and you try to call value on it?)